drivers/gpio/gpio_nrf5.c

```diff
@@ -98,8 +100,13 @@ struct gpio_nrf5_data {
 	((volatile struct _gpiote *)(DEV_GPIO_CFG(dev))->gpiote_base_addr)
 
 
-#define GPIO_SENSE_DISABLE    (GPIO_PIN_CNF_SENSE_Disabled << GPIO_PIN_CNF_SENSE_Pos)
-#define GPIO_SENSE_ENABLE     (GPIO_PIN_CNF_SENSE_Enabled << GPIO_PIN_CNF_SENSE_Pos)
+#define GPIO_SENSE_DISABLE	(GPIO_PIN_CNF_SENSE_Disabled	\
```

Comment:
Indentation and line split, do it after an operator:

#define GPIO_SENSE_DISABLE	(GPIO_PIN_CNF_SENSE_Disabled << \
                                                         GPIO_PIN_CNF_SENSE_Pos)